var/spack/repos/builtin/packages/py-ipython/package.py

```diff
@@ -25,11 +26,15 @@ class PyIpython(PythonPackage):
     depends_on('py-pathlib2', type=('build', 'run'), when="^python@:3.3")
     depends_on('py-pygments',                   type=('build', 'run'))
     depends_on('py-pickleshare',                type=('build', 'run'))
-    depends_on('py-simplegeneric@0.8:',         type=('build', 'run'))
-    depends_on('py-prompt-toolkit@1.0.4:1.999', when='@:7.0.0', type=('build', 'run'))
-    depends_on('py-prompt-toolkit@2.0.0:2.999', when='@7.0.0:', type=('build', 'run'))
+    depends_on('py-simplegeneric@0.8:',         type=('build', 'run'), when='@:7.4.0')
```

Comment:
```suggestion
    depends_on('py-simplegeneric@0.8:',         type=('build', 'run'), when='@:6')
```
Looks like it isn't needed for 7.3.0 either. If you want to be a hero you can figure out exactly which version dropped it.

Reply:
Easy - it's 7.1.0 according to their github.